{{cookiecutter.repo_name}}/app/scripts/router.js

```diff
@@ -47,6 +53,12 @@ app.config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $ur
             bodyClasses: 'four generic'
         }
     });
+
+    // Enable HTML5 mode (no hashbang)
+    $locationProvider.html5Mode(true);
```

Comment:
Since html5mode also requires handling server side and is affected by deployment strategy, so it make sense to have it an optional feature that get added to project if somebody say they to have it. I don't think it should be enabled by default.